extensions/PhabBugz/lib/Feed.pm

```diff
@@ -520,6 +521,53 @@ sub process_new_user {
     # Pre setup before querying DB
     my $old_user = set_phab_user();
 
+    # CHECK AND WARN FOR POSSIBLE USERNAME SQUATTING
+    INFO("Checking for username squatters");
+    my $dbh     = Bugzilla->dbh;
+    my $regexp  = $dbh->quote( ":?:" . quotemeta($phab_user->name) . "[[:>:]]" );
+    my $results = $dbh->selectall_arrayref( "
+        SELECT userid, login_name, realname
+          FROM profiles
+         WHERE userid != ? AND " . $dbh->sql_regexp( 'realname', $regexp ),
+        { Slice => {} },
+        $bug_user->id );
+    if (@$results) {
+        # The email client will display the Date: header in the desired timezone,
+        # so we can always use UTC here.
+        my $timestamp = Bugzilla->dbh->selectrow_array('SELECT LOCALTIMESTAMP(0)');
+        $timestamp = format_time($timestamp, '%a, %d %b %Y %T %z', 'UTC');
+
+        foreach my $row (@$results) {
+            WARN( "Possible username squatter: "
+                  . "phab user login: " . $phab_user->name
+                  . " phab user realname: " . $phab_user->realname
+                  . " bugzilla user id: " . $row->{userid}
+                  . " bugzilla login: " . $row->{login_name}
+                  . " bugzilla realname: " . $row->{realname} );
```

Comment:
The off-side formatting looks weird too. WARN behaves like warn and print, so you can use a list arguments.

```perl
WARN(
    'Possible username squatter: ',
    'phab user login: ' . $phab_user->name,
    ' phab user realname: ' . $phab_user->realname,
    ' bugzilla user id: ' . $row->{userid},
    ' bugzilla login: ' . $row->{login_name},
    ' bugzilla realname: ' . $row->{realname},
);
```

Reply:
Also perltidy/critic will complain about double quotes that don't need to be double quotes.